src/allotropy/parsers/appbio_quantstudio/appbio_quantstudio_structure.py

```diff
@@ -252,9 +252,9 @@ def create_generic(data: pd.Series[str]) -> WellItem:
 class Well:
     identifier: int
     items: dict[str, WellItem]
+    calculated_documents: list[CalculatedDocument]
```

Comment:
It seems like we only want to allow appending new documents after the class is instantiated, and that we always want to instantiate with an empty list. So how about:
`__calculated_documents: list[CalculatedDocument] = []`

... with a property a la how the two fields below are accessed, and a new method `add_calculated_document`?

Benefits:
1. Make it harder to accidentally reset `calculated_documents` 
2. No changing existing `Well` instantiations

Reply:
Yeah, the reason I avoided this was because it's inadvisable to use default `[]`, and I didn't want to do the extra work of dealing with if null, etc, etc. But you're right it's better to be fully correct than mostly correct, fixing.